### PR TITLE
Altered faiss installation comment

### PR DESCRIPTION
**Pull request by vishalsrao**

### CODE REVIEW

**Comment by lhoestq** [comment]
Hi ! Can you explain why this change is needed ?

File: src/datasets/search.py [hunk 1]
```diff
@@ -243,7 +243,7 @@ def __init__(
         self.faiss_index = custom_index
         if not _has_faiss:
             raise ImportError(
-                "You must install Faiss to use FaissIndex. To do so you can run `pip install faiss-cpu` or `pip install faiss-gpu`"
+                "You must install Faiss to use FaissIndex. To do so you can run `conda install -c pytorch faiss-cpu` or `conda install -c pytorch faiss-gpu`"
```
[the remark this answers is not in the document]
Comment (vishalsrao) [reply src/datasets/search.py:246]: Should we add this warning to the second line? "pip may not have the latest version of FAISS, and thus, some of the latest features and bug fixes may not be available."

[the remark this answers is not in the document]
Comment (lhoestq) [reply src/datasets/search.py:246]: sounds good !
```suggestion
                "You must install Faiss to use FaissIndex. To do so you can run `conda install -c pytorch faiss-cpu` or `conda install -c pytorch faiss-gpu`. "
                "A community supported package is also available on pypi: `pip install faiss-cpu` or `pip install faiss-gpu`. "
                "Note that pip may not have the latest version of FAISS, and thus, some of the latest features and bug fixes may not be available."
```

**Review by lhoestq — APPROVED** [review]
Thanks for the suggestion ! merging :)

**Pull request merged**